src/L1GovernorMetadataBridge.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {IGovernor} from "openzeppelin/governance/Governor.sol";
+import {IWormhole} from "wormhole/interfaces/IWormhole.sol";
+
+contract L1GovernorMetadataBridge {
+  /// @notice The governor where proposals are fetched and bridged.
+  IGovernor public governor;
+
+  /// @notice The Wormhole core contract used to relay messages.
+  IWormhole public coreBridge;
+
+  /// @notice The L2 governor metadata address where the message is sent on L2.
+  address public L2_GOVERNOR_ADDRESS;
+
+  /// @notice Indicates whether the contract has been initialized with the L2 governor metadata. It
+  /// can only be called once.
+  bool public INITIALIZED = false;
+
+  /// @notice A unique number used to send messages.
+  uint32 public nonce;
+
+  /// @notice The proposal id is an invalid proposal id.
+  error InvalidProposalId();
+
+  /// @param _governor The address of the L1 governor contract.
+  /// @param _core The address of the L1 core wormhole contract.
+  constructor(address _governor, address _core) {
+    governor = IGovernor(_governor);
+    coreBridge = IWormhole(_core);
+    nonce = 0;
```

Comment:
nit: not really needed—initializes to 0 by default

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/8/commits/d4a00f358908af70cf6016252ee2b4dab019d2f2